### DISCUSSION TITLE
No Longer will download "Unexpected property: 'incognito'" Error

### DISCUSSION THREAD

**Discussion by Kevski62** [aclap-dev/video-downloadhelper#308] [Help and Q&A] [answered]
### Your question

Hi - Everything has been fine up to now. Downloads will no longer trigger, despite VDH seeing files to download. Error message is:
_Error in invocation of downloads.download(downloads.DownloadOptions options, optional function callback): Error at parameter 'options': Unexpected property: 'incognito'_
**Full details:**
TypeError: Error in invocation of downloads.download(downloads.DownloadOptions options, optional function callback): Error at parameter 'options': Unexpected property: 'incognito'.
    at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:7834
    at new Promise (<anonymous>)
    at Proxy.<anonymous> (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:7515)
    at Object.apply (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:7916)
    at R2 (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:9777)
    at async ri (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:13229)

Any help or fixes appreciated.
Thanks

### Addon & CoApp version and details

Video DownloadHelper 8.2.2.8 (stable)
Video DownloadHelper
Version 8.2.2.8 (stable)
Browser locale: en-GB
Production build
Built on 2024-03-18
Build options: browser=chrome, linuxlic=false, noyt=true, target=google

**Comment by GrampaWildWilly** [comment]
Are you using incognito mode?  There is a setting that permits that during installation.  I believe you can go into the browser settings after you install VDH & enable that permission.  I've always permitted it during installation.  Also, I don't see any information on your CoApp.  If you're not using the CoApp, it's possible, not probable, just possible, that the CoApp might be needed here.  But since you haven't provided any URLs, I can't say that for certain.

**Reply by Kevski62** [reply]
Thanks for picking this up - No, I am not using incognito mode and I have tried reinstalling and enabling the permission, with no success. Just for info, the extension was working perfectly using the chrome browser to download across multiple sites with no issues until this evening. I have completed another fresh install and have added the CoApp this time. If I use the CoApp, it appears that the extension does now work again on the same range of domains that worked previously, so in effect, problem solved.
The only issue is that I have no idea what caused the issue in the first place . . . 
Thanks for your help though 👌👌😁

**Comment by ApexAlphaSigma** [comment]
Getting the same message. It was fine and all of the sudden got this error message. Tried uninstalling and reinstalling but still getting the message.

**Reply by roocketbunny** [reply]
Hi how to solve this? it's appear the same for me.

**Reply by ApexAlphaSigma** [reply]
No clue. Like I said Re and uninstalling doesn't do any good.

**Reply by GrampaWildWilly** [reply]
Guys, if you want help, you need to post a proper problem report, like @Kevski62 did at the top of this discussion.  Do be sure to include URLs of web pages on which problematic videos are hosted.  Meanwhile @Kevski62 solved his problem by installing the CoApp.  Perhaps that would work for you as well.  His problem report pointed up the fact that he initially did not have the CoApp installed at all.  Once he installed it, his problem was solved.  Since you guys haven't provided proper problem reports, I can't tell if that would be a relevant move for you.

**Comment by thewritingking** [comment]
I am getting the same error. Nothing helps. Not in incognito mode.

**Reply by GrampaWildWilly** [reply]
Ditto.

**Comment by paulrouget** [comment, marked as answer]
I broke this. My bad. I'm publishing a fix immediately. It's gonna take ~24 hours or so for Google to update the addon.

Like @Kevski62 said, you can use the CoApp download option in the meantime.

**Reply by Kevski62** [reply]
Thanks @paulrouget - I guessed it was an update of some sort that had caused the issue. Great to have such responsive support. 👍👍 Will give it a couple of days and switch back to browser mode to see if it works then.
